conf/webpack/Base.js

```diff
@@ -85,7 +85,7 @@ class WebpackBaseConfig {
 						loader: 'file-loader'
 					},
 					{
-						test: /^.((?!cssmodule).)*\.(sass|scss)$/,
+						test: /^.((?!cssmodule).)*\.(sass|scss|css)$/,
```

Comment:
Are we using any css files? It doesn't look like you added any.

Reply:
I had added some, and then removed.  Left it in there to save headaches in the future (if any react components need css).  Can remove but don't think it hurts.